src/adapters/node_adapter.js

```diff
@@ -173,7 +174,7 @@ var NodeAdapter = Class({ className: 'NodeAdapter',
           headers['Content-Disposition'] = 'attachment; filename=f.txt';
         }
 
-        headers['Content-Length'] = new Buffer(body, 'utf8').length.toString();
+        headers['Content-Length'] = Buffer.from(body, 'utf8').length.toString();
         headers['Connection'] = 'close';
```

Comment:
The merge created several conflicts, but this one was the only meaningful one.
It's a revert from https://github.com/faye/faye/issues/420 we decided to do in our fork because of issues we faced during a version upgrade (we experienced "WebSocket connection to 'ws://domain-test.ws.local.dimelo.info:9290/faye' failed: Error during WebSocket handshake: Unexpected response code: 502" errors, RCED reference ticket was RD-8061).
I kept our version keeping the `Connection: close` header.

Reply:
Ok sounds good